base/CMakeLists.txt

```diff
@@ -329,7 +329,6 @@ find_file(XERCES_JAR
 
 find_file(XML_COMMONS_APIS_JAR
     NAMES
-        xml-commons-apis.jar
         xml-apis.jar
     PATHS
         /usr/share/java
```

Comment:
I think this whole thing can be removed too. Feel free to update the patch, or leave it for a later cleanup. No need for re-review. Thanks!

Reply:
I'm very much a "clean it up right away" kinda guy :+1: